apps/admin-portal/src/pages/claim-dialects.tsx

```diff
@@ -98,14 +120,25 @@ export const ClaimDialectsPage = (): React.ReactElement => {
                 <ListLayout
                     advancedSearch={
                         <DialectSearch onFilter={ (query) => {
-                            getDialect(null, null, null, query);
+                            // getDialect(null, null, null, query);
```

Comment:
Can we remove this commented line ? If it's something we will use later. Add a TODO